test/e2e/framework/volume/fixtures.go

```diff
@@ -564,7 +565,7 @@ func testVolumeContent(client clientset.Interface, pod *v1.Pod, fsGroup *int64,
 // Multiple Tests can be specified to mount multiple volumes to a single
 // pod.
 func TestVolumeClient(client clientset.Interface, config TestConfig, fsGroup *int64, fsType string, tests []Test) {
-	clientPod, err := runVolumeTesterPod(client, config, "client", fsGroup, tests)
+	clientPod, err := runVolumeTesterPod(client, config, "client", fsGroup, false, tests)
```

Comment:
How come this one is false and the other is true?

Can we also make privileged a configurable parameter? I would like to be able to have most e2es not use privileged pods if the system doesn't support SELinux: https://github.com/kubernetes/kubernetes/issues/80186

Reply:
> Can we also make privileged a configurable parameter?

So I figured this is a bit tricky to solve. These are the possible solutions that I thought about and their problems:

1. Make pods privileged:
	* We don't want this, specially where they are not required (on non-SELinux systems)
2. Make pods unprivileged and add label `svirt_sandbox_file_t` to node dirs where pods manipulate data (currently it's `/tmp` for hostPath)
	* Need to manually `chcon -R -t svirt_sandbox_file_t /dir_where_container_writes_files` in the host where test runs
3. Detect if node, where test is running, has SELinux enabled and make the pod privileged
	* I attempted this one, but it's tricky because we don't always know the node where the pod is going to be scheduled (so we could send `getenforcea command there and 
4. Have an env variable that toggles privileged
	* Not sure if this is a good idea, it needs to be toggled manually
        * It's not node-specific

Are there other alternatives?